applications/vessel-history/src/features/profile/components/activity/ActivityModalContentDetailsPortVisit.tsx

```diff
@@ -75,10 +78,22 @@ const ActivityModalContentDetailsPortVisit: React.FC<ActivityModalContentProps>
         <ActivityModalContentField
           label={t('event.confidence', 'Confidence')}
           value={confidence ?? DEFAULT_EMPTY_VALUE}
+          help={help}
         />
         <ActivityModalContentField label={t('event.psma', 'PSMA')} value={psmaDescription} />
       </div>
       <ActivityModalContentDetails event={event} />
+      {[2,3,4].includes(confidenceLevel) && (
+        <div className={styles.help}>
+            <p>{t(`event.confidenceHelp.help${confidenceLevel.toString()}` as any)}</p>
+            <ol>
```

Comment:
This should be `ul` instead of `ol`, right?

Reply:
the example are numbers, not points